src/components/App/App.js

```diff
@@ -2,17 +2,27 @@ import { useState } from 'react'
 import movieData from '../../movieData'
 import MoviesContainer from '../MoviesContainer/MoviesContainer'
 import Header from '../Header/Header'
+import SelectedMoviesContainer from '../SelectedMoviesContainer/SelectedMoviesContainer'
 import './App.css'
 
 
 const App = () => {
   const [movies, setMovies] = useState(movieData)
   // console.log('movieData', movieData)
+  const [selectedMovie, setSelectedMovie] = useState('')
+
+  const handleMovieCardClick = (movieId) => {
```

Comment:
Love seeing this handler function declared in app and passed as a prop to be invoked elsewhere.  Nice flow here!